src/components/Header/Header.css

```diff
@@ -150,7 +151,7 @@
   z-index: -1;
 }
 
-.guidebook_open_help .icon {
```

Comment:
why replace .icon with img ... I don't think that will work?

Reply:
this one isn't an icon